src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/AuditApiAction.java

```diff
@@ -102,12 +102,6 @@
  * [{"op": "replace", "path": "/config/compliance/internal_config", "value": "true"}]
  */
 public class AuditApiAction extends PatchableResourceApiAction {
-    private static final List<Route> routes = ImmutableList.of(
```

Comment:
Please squash these changes to a corresponding cherry-picked commit. The PR should consist only of cherry-picked commits.